tests/formattedcode/data/spdx/license_known/expected.tv

```diff
@@ -1,35 +1,36 @@
-# Document Information
+## Document Information
 SPDXVersion: SPDX-2.2
 DataLicense: CC0-1.0
-DocumentNamespace: http://spdx.org/spdxdocs/scan
-DocumentName: SPDX Document created by ScanCode Toolkit
-LicenseListVersion: 3.20
 SPDXID: SPDXRef-DOCUMENT
+DocumentName: SPDX Document created by ScanCode Toolkit
+DocumentNamespace: http://spdx.org/spdxdocs/scan
 DocumentComment: <text>Generated with ScanCode and provided on an "AS IS" BASIS, WITHOUT WARRANTIES
 OR CONDITIONS OF ANY KIND, either express or implied. No content created from
 ScanCode should be considered or used as legal advice. Consult an Attorney
 for any legal advice.
 ScanCode is a free software code scanning tool from nexB Inc. and others.
-Visit https://github.com/nexB/scancode-toolkit/ for support and download.</text>
-# Creation Info
-# Package
+Visit https://github.com/nexB/scancode-toolkit/ for support and download.
+SPDX License List: 3.21</text>
+## Creation Information
+## Package Information
 PackageName: scan
 SPDXID: SPDXRef-001
 PackageDownloadLocation: NOASSERTION
+FilesAnalyzed: True
 PackageVerificationCode: 517c26f04b5a3edaaa0c17bf1a6571953209e209
-PackageLicenseDeclared: NOASSERTION
```

Comment:
Why not keeping the original ordering? Declared before Concluded is the right natural ordering we have until now?